### PR TITLE
Implement ember-paper

### PR DESCRIPTION
**Pull request by gschorkopf**
📱 🎉 

Working list of TODO items:

- [x] All commented out code removed
- [x] All quotes are single-quotes where appropriate
- [x] Add extra space to EOF diffs
- [x] Clean up footer button styles, footer padding
- [x] Perhaps the "play" button should go to `rounds.new`
- [ ] Add styling to `rounds` table and `turns` scorekeeping pages that look rough after removing bootstrap
- [ ] Fix / stylize the "New Round" form so it's back to functional (user paper-chips for users?)
- [ ] Fix the autocomplete for users/courses so it's back to functional

### CODE REVIEW

**Comment by gschorkopf** [comment]
See code comments here: https://github.com/frolfr/frolfr-web/pull/3

File: app/components/round-form.js [hunk 1]
```diff
@@ -59,6 +86,20 @@ export default Ember.Component.extend({
     save() {
       this.get('model').save()
       .then((round) => this.sendAction('didSave', round));
+    },
+
+    closeFriendsDialog(event) {
+      if(event === 'save') {
+        // do ssomething
```
Comment (gschorkopf) [inline app/components/round-form.js]: What's going on here?

File: app/transitions.js [hunk 2]
```diff
@@ -54,4 +54,15 @@ export default function(){
     this.use('toLeft'),
     this.reverse('toRight')
   );
+
+  this.transition(
+  this.hasClass('liquid-dialog-container'),
+  this.use('explode', {
```
Comment (gschorkopf) [inline app/transitions.js]: 💥

File: app/styles/app.scss [hunk 3]
```diff
@@ -1,5 +1,7 @@
 @import 'variables';
 @import 'ember-paper';
+@import "ember-paper-expansion-panel";
```
Comment (gschorkopf) [inline app/styles/app.scss]: Where'd this come from? Are there other things we can import (like table styles)?

File: app/components/round-form.js (hunk 1 above)
Comment (ultimatemonty) [reply app/components/round-form.js]: Placeholder to make sure things worked like I wanted. Will implement when the "who are you playing with" dialog is implemented

File: app/styles/app.scss (hunk 3 above)
Comment (ultimatemonty) [reply app/styles/app.scss]: Ugh I missed removing that. I toyed around with expansion panels in the select course dialog before sticking with tabs

Comment (ultimatemonty) [reply app/styles/app.scss]: Table styles should be there by default. See http://miguelcobain.github.io/ember-paper/#/typography

Comment (gschorkopf) [reply app/styles/app.scss]: 👍 I was gonna say, I got `Error: File to import not found or unreadable: ember-paper-expansion-panel.`

Also... if the table styles should be there, I wonder why they're not?

File: app/services/ui.js [hunk 4]
```diff
@@ -6,13 +6,13 @@ export default Service.extend({
   router: service(),
 
   isFabVisible: computed('router.currentRouteName', function() {
-    const disabledRoutes = [ 'rounds.new', 'rounds.current' ];
+    const disabledRoutes = [ 'rounds.new', 'round.turns' ];
```
Comment (gschorkopf) [inline app/services/ui.js]: Neat!

File: app/components/round-form.js [hunk 5]
```diff
@@ -4,8 +4,9 @@ const { Component, computed, inject } = Ember;
 const { service } = inject;
 
 export default Component.extend({
-  classNames: ['round-form'],
+  classNames: ['round-form', 'flex-100'],
```
Comment (gschorkopf) [inline app/components/round-form.js]: What's `flex-100`?

**Comment by gschorkopf** [comment]
Will rebase.

**Comment by gschorkopf** [comment]
I don't know how valuable the `friends` route is going to be without being able to sign in. Maybe let's move friends to a separate branch?

**Comment by gschorkopf** [comment]
Rebased delicately, based on the eye test looks like we're good.

**Pull request merged**